Project-I-Social-Card/social-card/src/components/CardComponents/CardBanner.js

```diff
@@ -1,2 +1,9 @@
 import React from 'react';
 import './Card.css';
+
+const CardBanner = () => {
+	return (
+		<img class="banner-img" src=" https://tk-assets.lambdaschool.com/fcd75197-7d12-46ec-bc9e-4130f34822fa_reactbackground.png"alt=""/>
```

Comment:
The *class* attribute should be *src* instead.

Reply:
Also, *alt* should contain a very brief description of the image or the console will show a warning.